CMakeLists.txt

```diff
@@ -72,6 +72,8 @@ option(ENABLE_CODE_TIMERS "Enable code timers" OFF)
 
 option(ENABLE_TRACY "Enable Tracy Profiling" OFF)
 
+option(ENABLE_LIBCXX "Disable using libc++" ON)
```

Comment:
This comment is confusing. The variable name indicates setting this variable to ON means to enable lib++ but comment says **disable**

Reply:
Thanks! I renamed the variable and not the comment :)